loader/icd.c

```diff
@@ -26,13 +26,24 @@
 #include <string.h>
 
 KHRicdVendor *khrIcdVendors = NULL;
+int khrEnableTrace = 0;
+
 #if defined(CL_ENABLE_LAYERS)
 struct KHRLayer *khrFirstLayer = NULL;
 #endif // defined(CL_ENABLE_LAYERS)
 
 // entrypoint to initialize the ICD and add all vendors
 void khrIcdInitialize(void)
 {
+    char *enableTrace = khrIcd_getenv("OCL_ICD_ENABLE_TRACE");
+    if (enableTrace && (strcmp(enableTrace, "True") == 0 ||
+            strcmp(enableTrace, "true") == 0 ||
+            strcmp(enableTrace, "T") == 0 ||
+            strcmp(enableTrace, "1") == 0))
+    {
+	khrEnableTrace = 1;
+    }
+
```

Comment:
This should go in its own subroutine. also, line 44 was indented with a tabulation.

Reply:
I added on function named khrIcdInitializeTrace for it.